src/reporting/interfaces/mappingInterfaces/GroupProperties.ts

```diff
@@ -0,0 +1,203 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { PagedResponseLinks } from "../Links";
+
+/**
+ * Defines a target ECProperty for a Group. This is used to determine what properties will be read from the query response of a Group.
+ * @export
+ * @interface ECProperty
+ */
+export interface ECProperty {
+  /**
+   * The target ECSchema.
+   * @type {string}
+   * @memberof ECProperty
+   */
+  ecSchemaName: string;
+  /**
+   * The target ECClass.
+   * @type {string}
+   * @memberof ECProperty
+   */
+  ecClassName: string;
+  /**
+   * The target ECProperty.
+   * @type {string}
+   * @memberof ECProperty
+   */
+  ecPropertyName: string;
+  /**
+   * The type of the target ECProperty.
+   * @type {string}
+   * @memberof ECProperty
+   */
+  ecPropertyType: DataType;
+}
+
+/**
+ * Defines a one-to-many property mapping to a set of ECProperties for a Group - to be used when a Group query will return data from multiple ECClasses.
```

Comment:
This comment needs to be rewritten. Go through documentation here (https://developer.bentley.com/apis/insights/operations/get-groupproperty/) and update the comment. The part that this GroupProperty defines one-to-many property mapping is kind of misleading.

Reply:
changed to ``` Defines a mapping for a group and some number of distinct ECProperties ```